python/benchmarks/bi_benchmarks.py

```diff
@@ -0,0 +1,168 @@
+"""
+Copyright 2023 Man Group Operations Limited
+
+Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
+
+As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+"""
+import os
+from pathlib import Path
+from arcticdb import Arctic
+from arcticdb.version_store.processing import QueryBuilder
+
+from .common import *
+
+
+def get_query_groupby_city_count_all(q):
+    return q.groupby("City").agg({"Keyword": "count"})
```

Comment:
nit: it would be better programming practice to construct the `QueryBuilder` in these methods and then return it, rather than taking a parameter.

Reply:
Agree, but in this particular case I use those methods to pass QueryBuilder or DataFrame, so that I can use same methods to work with arctic and dataframes (pandas (see the setup cache)

Reply:
But you have point as this si not obvious, I will add type hints here

Reply:
Ah I see, makes sense